src/components/forms/PlanForm/helpers.ts

```diff
@@ -880,8 +885,8 @@ export function getPlanFormValues(planObject: PlanDefinition): PlanFormFields {
     interventionType,
     jurisdictions: planObject.jurisdiction.map(e => ({
       id: e.code,
-      name: e.code,
-    })) /** a little cheating: since we dnt have the name yet, we just use code */,
+      name: '',
```

Comment:
This makes the name by default `''`.  Why?

Reply:
This helps avoid replacing of `name` or `title` with the code if a user tries to do edits before the  fetch of jurisdiction name (done by another [component](https://github.com/onaio/reveal-frontend/blob/master/src/containers/pages/InterventionPlan/UpdatePlan/PlanLocationNames/index.tsx)) is complete.